ops/mainnet/prod/backend/main.tf

```diff
@@ -43,7 +43,7 @@ module "cartographer_db" {
     Domain      = var.domain
   }
 
-  parameter_group_name = "default.postgres14"
+  parameter_group_name = "rds-postgres"
```

Comment:
Hmmm, this needs to depend on the other module, otherwise if one gets created before the other, this might fail.

What you can do is get the output of the `aws_db_parameter_group` and then use it in `main.tf`

Something like
```
# ops/modules/db/outputs.tf

output "rds_parameter_group" {
   vault = aws_db_parameter_group.rds_postgres.name
}

# ops/mainnet/prod/main.tf

parameter_group_name = module.db.parameter_group_name
```

Reply:
Roger that